test/integration/filters/BUILD

```diff
@@ -93,3 +99,17 @@ envoy_cc_test_library(
         "//test/test_common:utility_lib",
     ],
 )
+
+envoy_cc_test_library(
+    name = "sleeping_filter_lib",
+    srcs = [
+        "sleeping_filter.cc",
+    ],
```

Comment:
nit: I think `srcs = ["sleeping_filter.cc"],` is preferred, looking at existing BUILDs.

Reply:
Addressed in 8bd4def.